src/main/scala/nl/knaw/dans/easy/stage/dataset/Util.scala

```diff
@@ -67,6 +67,25 @@ object Util {
     } yield audience.text
   }
 
+  def readFileType(filePath: String)(implicit s: Settings): Try[Option[String]] = Try {
+    val filetype = for {
+      file <- loadBagXML("metadata/files.xml") \\ "files" \ "file"
+      if (file \ "@filepath").text == filePath
+      filetype <- file \ "type"
+    } yield filetype
+    if (filetype.size == 1) Option(filetype.head.text)
+    else None
+  }
+
+  def isAVType(filePath: String)(implicit s: Settings): Try[Boolean] = {
+    readFileType(filePath).flatMap {
+      case Some("http://schema.org/AudioObject") => Success(true)
```

Comment:
Combine this and the next pattern match, since they give the same output:

```scala
case Some("http://schema.org/AudioObject") | Some("http://schema.org/VideoObject") => Success(true)
```